apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue

```diff
@@ -178,7 +180,23 @@ export default defineComponent<Props>({
         const { referenceStoreState } = useReferenceStore();
 
         /* more options */
-        const handleUpdateSchemaProperties = (properties: string[]) => {
+        const updateSchemaFormDataBySchemaProperties = (oldSchemaProperties: string[], newSchemaProperties: string[]) => {
+            if (oldSchemaProperties.length > newSchemaProperties.length) { // delete case
+                const deletedProperties = xor(oldSchemaProperties, newSchemaProperties);
+                deletedProperties.forEach((propertyName) => {
+                    delete state.schemaFormData[propertyName];
+                });
+            } else if (oldSchemaProperties.length < newSchemaProperties.length) { // add case
+                const addedProperties = xor(oldSchemaProperties, newSchemaProperties);
+                addedProperties.forEach((propertyName) => {
+                    state.schemaFormData[propertyName] = undefined;
+                });
+            }
```

Comment:
1. check added / deleted property 
2. update `schemaFormData` (This triggers validation within the `JsonSchemaForm` component.)